### PR TITLE
Add examples for master-minion path-regex

### PR DESCRIPTION
**Pull request by jjngx**
### Proposed changes

Update documentation and add examples of how to use Master-Minion Ingresses with `path-regex` annotation.

### Checklist

Before creating a PR, run through this checklist and mark each as complete.

- [X] I have read the [CONTRIBUTING](https://github.com/nginxinc/kubernetes-ingress/blob/main/CONTRIBUTING.md) doc
- [ ] I have added tests that prove my fix is effective or that my feature works
- [X] I have checked that all unit tests pass after adding my changes
- [X] I have updated necessary documentation
- [X] I have rebased my branch onto main
- [X] I will ensure my PR is targeting the main branch and pulling from my branch from my own fork

### CODE REVIEW

File: docs/content/configuration/ingress-resources/advanced-configuration-with-annotations.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ The table below summarizes the available annotations.
...
-|``nginx.org/path-regex`` | N/A | Enables regular expression modifiers for [location](https://nginx.org/en/docs/http/ngx_http_core_module.html#location) directive. You can specify one of these values: "case_sensitive", "case_insensitive" or "exact". The annotation is applied to the entire Ingress resource and its paths. |  |  [Path Regex](https://github.com/nginxinc/kubernetes-ingress/tree/examples/ingress-resources/path-regex). |
+|``nginx.org/path-regex`` | N/A | Enables regular expression modifiers for [location](https://nginx.org/en/docs/http/ngx_http_core_module.html#location) directive. You can specify one of these values: "case_sensitive", "case_insensitive", or "exact". The annotation is applied to the entire Ingress resource and its paths. While using Master and Minion Ingresses i.e. Mergeable Ingresses, this annotation can be specified on Minion types. The `path-regex` annotation specified on Master is ignored, and has no effect on paths defined on Minions.   |  |  [Path Regex](https://github.com/nginxinc/kubernetes-ingress/tree/examples/ingress-resources/path-regex). |
```
Comment (shaun-nx) [inline docs/content/configuration/ingress-resources/advanced-configuration-with-annotations.md:115]:
```suggestion
|``nginx.org/path-regex`` | N/A | Enables regular expression modifiers for [location](https://nginx.org/en/docs/http/ngx_http_core_module.html#location) directive. You can specify one of these values: "case_sensitive", "case_insensitive", or "exact". The annotation is applied to the entire Ingress resource and its paths. While using Master and Minion Ingresses i.e. Mergeable Ingresses, this annotation can be specified on Minion types. The `path-regex` annotation specified on Master is ignored, and has no effect on paths defined on Minions.   |  N/A |  [Path Regex](https://github.com/nginxinc/kubernetes-ingress/tree/examples/ingress-resources/path-regex). |
```
We typically use `N/A` in scenarios where we don't have a default.

Comment (jjngx) [reply docs/content/configuration/ingress-resources/advanced-configuration-with-annotations.md]: updated, thx!

**Review by shaun-nx — APPROVED** [review]

File: docs/content/configuration/ingress-resources/advanced-configuration-with-annotations.md [hunk 2]
```diff
@@ -112,7 +112,7 @@ The table below summarizes the available annotations.
...
-|``nginx.org/path-regex`` | N/A | Enables regular expression modifiers for [location](https://nginx.org/en/docs/http/ngx_http_core_module.html#location) directive. You can specify one of these values: "case_sensitive", "case_insensitive" or "exact". The annotation is applied to the entire Ingress resource and its paths. |  |  [Path Regex](https://github.com/nginxinc/kubernetes-ingress/tree/examples/ingress-resources/path-regex). |
+|``nginx.org/path-regex`` | N/A | Enables regular expression modifiers for [location](https://nginx.org/en/docs/http/ngx_http_core_module.html#location) directive. You can specify one of these values: "case_sensitive", "case_insensitive", or "exact". The annotation is applied to the entire Ingress resource and its paths. While using Master and Minion Ingresses i.e. Mergeable Ingresses, this annotation can be specified on Minion types. The `path-regex` annotation specified on Master is ignored, and has no effect on paths defined on Minions.   | N/A |  [Path Regex](https://github.com/nginxinc/kubernetes-ingress/tree/examples/ingress-resources/path-regex). |
```
Comment (brianehlert) [inline docs/content/configuration/ingress-resources/advanced-configuration-with-annotations.md:115]: Enables regular expression modifiers for Ingress path parameter.  This translates to the NGINX [location](https://nginx.org/en/docs/http/ngx_http_core_module.html#location) directive.

**Review by brianehlert — COMMENTED** [review]
one comment.

File: docs/content/configuration/ingress-resources/advanced-configuration-with-annotations.md (hunk 2 above)
Comment (jjngx) [reply docs/content/configuration/ingress-resources/advanced-configuration-with-annotations.md:115]: @brianehlert this change is addressed in [this](https://github.com/nginxinc/kubernetes-ingress/pull/4247) PR

**Pull request merged**